Chat/ClientApp/src/core/sideEffects.ts

```diff
@@ -442,7 +538,7 @@ const addThreadMemberHelper = async (threadId: string, user: User, dispatch: Dis
 const getThreadMembersHelper = async (chatThreadClient: ChatThreadClient) => {
```

Comment:
can we confirm if we really even get getThreadMembers as well?

Reply:
we only used this helper in one spot so I removed the function and just moved the lines over to its one usage area